lib/Doctrine/ORM/Query/Expr/Func.php

```diff
@@ -73,6 +75,10 @@ public function getArguments()
      */
     public function __toString()
     {
+        if (count($this->arguments) === 0) {
+            return $this->name . '(NULL)';
```

Comment:
Wait so `IN` is considered a function :thinking: :thinking: :thinking: ? I mean it has a left operand…

Reply:
Yes, that's what I discovered 😅

Reply:
This makes it a bit scary… touching such a generic method looks like it could have unforeseen repercussions.